cdap-app-templates/cdap-etl/cdap-etl-lib/src/test/java/co/cask/cdap/etl/transform/ScriptTransformTest.java

```diff
@@ -197,6 +197,8 @@ public void testComplex() throws Exception {
       "  var e = input.inner1.list[0].e;\n" +
       "  var val = power(pi.val, 3) + power(e.val, 2);\n" +
       "  print(pi); print(e);\n" +
+      "  metrics.count(\"script.transform.count\", 1);\n" +
```

Comment:
can we check that this was emitted somehow? I think there are some tests that use a mock metrics.